### PR TITLE
minor typo thing fixes

### PR DESCRIPTION
**Pull request by anconfuzedrock**
yea thats it

### CODE REVIEW

File: code/game/objects/items/stacks/rods.dm [hunk 1]
```diff
@@ -6,7 +6,7 @@ var/global/list/datum/stack_recipe/rod_recipes = list ( \
 
 /obj/item/stack/rods
 	name = "metal rod"
-	desc = "Some rods. Can be used for building, or something."
+	desc = "Some rods. Can be used for building or something."
```
Comment (ShadowDeath6) [inline code/game/objects/items/stacks/rods.dm]: personally the comma gives it an artistic touch and a suspenseful pause

Comment (CosmicScientist) [inline code/game/objects/items/stacks/rods.dm]: As an armchair nut for grammar and spelling, I would say no comma is correct however the pause hints at greater purpose.

Either or, neither are more right.

Comment (anconfuzedrock) [inline code/game/objects/items/stacks/rods.dm]: As rock, you dont need to be an armchair nut to fucking know that the , isnt needed I dknt think the description of rods needs to be deep its fucking rods

**Comment by bgobandit** [comment]
rock got his period

**Comment by optimumtact** [comment]
Can you stop making pr's and then closing them?

**Comment by anconfuzedrock** [comment]
I closed this? when did I close this? was I confused?

**Comment by anconfuzedrock** [comment]
sorry about that

**Comment by bgobandit** [comment]
you probably hit comment + close

**Pull request merged**